core/server/server.js

```diff
@@ -15,7 +15,10 @@ const GithubConstellation = require('../../services/github/github-constellation'
 const suggest = require('../../services/suggest')
 const { loadServiceClasses } = require('../base-service/loader')
 const { makeSend } = require('../base-service/legacy-result-sender')
-const { handleRequest } = require('../base-service/legacy-request-handler')
+const {
+  handleRequest,
+  clearRequestCache,
+} = require('../base-service/legacy-request-handler')
```

Comment:
Could you do a rebase? Seems like this is missing 2d9ac83bb4fc6e19500aca62c376ed7ceba1a6f0, and we wouldn't want to add these back and re-introduce the issue

Reply:
This will also be needed to fix the service tests